packages/dev/s2-docs/pages/react-aria/Toast.mdx

```diff
@@ -0,0 +1,210 @@
+</InlineAlert>
+
+## Auto-dismiss
+
+Use the `timeout` option to automatically dismiss toasts after a period of time. For accessibility, toasts should have a minimum timeout of **5 seconds**. Timers automatically pause when the user focuses or hovers over a toast.
+
+```tsx render hideImports
+"use client";
+import {queue} from 'vanilla-starter/Toast';
+import {Button} from 'vanilla-starter/Button';
+
+function Example() {
+  return (
+    <Button
+      ///- begin highlight -///
+      onPress={() => queue.add(
+        {title: 'File has been saved!'},
+        {timeout: 5000}
+      )}
+      ///- end highlight -///
+    >
+      Save file
+    </Button>
+  );
+}
+```
+
+<InlineAlert variant="notice">
+  <Heading>Accessibility</Heading>
+  <Content>Only auto-dismiss toasts when the information is not critical, or may be found elsewhere. Some users may require additional time to read toasts, and screen zoom users may miss them entirely.</Content>
+</InlineAlert>
+
+## Programmatic dismissal
+
+Toasts can be programmatically dismissed using the key returned from `queue.add()`. This is useful when a toast becomes irrelevant before the user manually closes it.
+
+```tsx render hideImports
+"use client";
+import {queue} from 'vanilla-starter/Toast';
+import {Button} from 'vanilla-starter/Button';
+import {useState} from 'react';
+
+function Example() {
+  let [toastKey, setToastKey] = useState(null);
+
+  return (
+    <Button
+      ///- begin highlight -///
+      onPress={() => {
+        if (!toastKey) {
+          setToastKey(queue.add(
+            {title: 'Processing...'},
+            {onClose: () => setToastKey(null)}
+          ));
+        } else {
+          queue.close(toastKey);
+        }
+      }}
+      ///- end highlight -///
+    >
+      {toastKey ? 'Cancel' : 'Process'}
+    </Button>
+  );
+}
+```
+
+## Accessibility
+
+Toast regions are [landmark regions](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/) that can be navigated using <Keyboard>F6</Keyboard> to move forward and <Keyboard>Shift</Keyboard> + <Keyboard>F6</Keyboard> to move backward. This provides an easy way for keyboard users to jump to toasts from anywhere in the app.
+
+When a toast is closed, focus moves to the next toast if any. When the last toast is closed, focus is restored to where it was before.
+
+## API
+
+<Anatomy role="img" aria-label="Toast anatomy diagram, showing the toast's title and close button within the toast region." />
+
+A `<ToastRegion>` is an ARIA landmark region labeled "Notifications" by default. It accepts a render function that receives each visible toast and renders a `<Toast>` component. Each toast is a non-modal ARIA [alertdialog](https://www.w3.org/WAI/ARIA/apg/patterns/alertdialog/) containing `<ToastContent>` and a close button.
```

Comment:
We removed these descriptions in all of our other component docs. I think this is covered by the Accessibility section above.